### DISCUSSION TITLE
Expansion of variable inside of ``

### DISCUSSION THREAD

**Discussion by ckrenslehner** [casey/just#2433] [Q&A] [answered]
Hi!

Lets say I have the following:

``` 
#global 
config := "config/cliff.toml"
version := `git cliff -c {{ config }}`

# recipes..
```

But the variable `config` is not expanded in `version`. I cannot find anything in the documentation how to make this work. I use `nu` as the shell in the justfile btw.
Thanks for your help!

**Comment by laniakea64** [comment, marked as answer]
The [`shell()` function](https://just.systems/man/en/functions.html#external-commands) accepts a `just` expression for the command string:
```
config := "config/cliff.toml"
version := shell('git cliff -c ' + config)
```

**Reply by ckrenslehner** [reply]
Perfect, thanks! :-)
